Automattic/jetpack
# Social Image Generator: Reduce template image file size

While testing, the images in the template picker were loading slowly. They were all at least 100kb in size. Compressing them a bit and reducing the quality to 80% has them all <50kb, reducing the bundle size and making the loading much faster.





### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Enable SIG on a site
* Click the change template button
* With this change the previews should load a bit quicker, but look the same as without it.
